editor.planx.uk/src/pages/FlowEditor/components/Flow/FeedbackLog/FeedbackLog.tsx

```diff
@@ -52,6 +52,9 @@ export const FeedbackLog: React.FC<FeedbackLogProps> = ({ feedback }) => {
                   <TableCell sx={{ width: 100 }}>
                     <strong>Date</strong>
                   </TableCell>
+                  <TableCell sx={{ width: 140 }}>
+                    <strong>Feedback score</strong>
```

Comment:
nit: since we're already in the "Feedback log", can this heading simply be "Score" (these tables don't have tons of real estate to begin with!)